spec/swagger/definitions/get_reasons_responses.json

```diff
@@ -0,0 +1,44 @@
+{
+  "200": {
+    "type": "object",
+    "required": [
+      "data"
+    ],
+    "properties": {
+      "data": {
+        "type": "array",
+        "items": {
+          "$ref": "#/definitions/reason/Reason"
```

Comment:
there is no hint that reason is about  `move` here, neither in the path, and in the filename.